lib/processors/background/BackgroundProcessor.ts

```diff
@@ -283,4 +277,19 @@ export class BackgroundProcessor extends Processor {
     _benchmark.end('processFrameDelay');
     _benchmark.start('captureFrameDelay');
   }
+
+  /**
+   * Gets the dimensions of a frame buffer based on its type
+  */
+  private getFrameDimensions(buffer: OffscreenCanvas | HTMLCanvasElement | HTMLVideoElement | VideoFrame): { width: number, height: number } {
```

Comment:
Though this is already private, we prefix private members with an underscore (`_`) since this method still shows up for customers using JS.